Update FAQ for Identify Calls Not Updating Intercom Profiles (#7277)
segmentio/segment-docs: src/connections/destinations/catalog/actions-intercom-web/index.md

> ### Intercom does not support rETL event batching
> The Intercom (Web) Actions destination does not support the bulk contacts endpoint, and therefore is unable to support batching events in rETL.

Reviewer: ```suggestion
### Intercom does not support Reverse ETL event batching
The Intercom (Web) Actions destination does not support the bulk contacts endpoint, and therefore is unable to support batching events in Reverse ETL.
```

> ### Why are my Identify() calls not updating/creating Intercom profiles or showing users as leads/visitors?

Reviewer: ```suggestion
### Why are my Identify calls not updating or creating Intercom profiles, or not showing users as leads or visitors?
```

> Intercom requires requests to include user data/traits beyond `email` or `user_hash` to update/create profiles or change user status from leads/visitors. Without additional user data/traits, Intercom skips sending a "ping" requests assuming no changes were made to the user data.

Reviewer: ```suggestion
Intercom requires requests to include user data/traits beyond `email` or `user_hash` to update or create profiles and change user status from leads/visitors. Without additional user data/traits, Intercom assumes no changes were made to a user's data and does not send a "ping" request.
```

Example Scenarios

> * Doesn't Work:

Reviewer: ``` suggestion
In the following example, which only includes an `email` and `user_hash`, Intercom would not send a "ping" request and update the status of this user:
```

```
analytics.identify("123");

analytics.identify("123", { email: "example@domain.com" });

analytics.identify("123",{email: "example@domain.com"}, {
   integrations: {
    Intercom: {
      user_hash: "81b65b9abea0444437a5d92620f03acc33f04fabbc32da1e047260024f80566a"
    }
  }})
```

> * Works:
>   
> Adding a trait like `name` resolves the issue:

Reviewer: ```suggestion
However, in the following example that also contains the `name` trait, Intercom sends a "ping" request and updates the status of this user:
```

```
analytics.identify("123", {
   email: "example@domain.com",
   name: "John Doe"
}, {
   integrations: { Intercom: { user_hash: "hash" } }
});
```

> Always include a trait, such as `name` or if you don't have a trait to send with identify calls, just map Segment `timestamp` to Intercom `last_request_at` in Segment to Intercom.

Reviewer: ```suggestion
When sending calls to Intercom, always include a trait, like`name`. If you don't have a trait to send with Identify calls, map Segment's `timestamp` field to Intercom's `last_request_at` field.
```
